appveyor.yml

```diff
@@ -60,15 +70,15 @@ artifacts:
   - path: 'TestResult.xml'
 
 
-skip_commits:
-  files:
-    - Docs/
-    - .github/
-    - Artwork/
-    - '**/*.md'
-    - LICENSE.txt
-    - README.txt
-    - .gitignore
-    - source/Archive/
-    - Users/
+#skip_commits:
+#  files:
+#    - Docs/
+#    - .github/
+#    - Artwork/
+#    - '**/*.md'
```

Comment:
Should this one also be commented out?

Reply:
I've had to comment out the old excludes because it's causing GitHub to indefinitely wait for a status. I'm currently looking into the GitHub API to set the success status for these kinds of commits involving MD files and more

Reply:
Shallow clone being set to true prevented the install: git diff to work properly that's fixed. so the skip_commits: list isn't needed, only to reference which files should be checked for in a git diff